ios/Podfile.lock

```diff
@@ -2186,11 +2199,14 @@ PODS:
     - ReactCommon/turbomodule/bridging
     - ReactCommon/turbomodule/core
     - Yoga
-  - SDWebImage (5.19.7):
-    - SDWebImage/Core (= 5.19.7)
-  - SDWebImage/Core (5.19.7)
+  - SDWebImage (5.21.0):
+    - SDWebImage/Core (= 5.21.0)
+  - SDWebImage/Core (5.21.0)
   - SDWebImageSVGCoder (1.7.0):
     - SDWebImage/Core (~> 5.6)
+  - SDWebImageWebPCoder (0.14.6):
+    - libwebp (~> 1.0)
+    - SDWebImage/Core (~> 5.17)
```

Comment:
libwebp added in native test here:

https://github.com/expo/expo/pull/35802/files#diff-6614a21a4ad3fc08014eb274a80fd3d80d6adca7e8972df369f2eee07e0ad126R2184-R2186